.pre-commit-config.yaml

```diff
@@ -77,22 +77,18 @@ repos:
             - --quiet
 
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.4.7
+    rev: v0.4.10
     hooks:
       - id: ruff
-        types_or: [python, pyi, jupyter]
```

Comment:
Ruff isn't great at modifying Jupyter notebook files (as validated in my other repos), so I removed it from our list. If we end up wanting that functionality, I think the [nbQA](https://github.com/nbQA-dev/nbQA) package would be a far better choice.